docs/configuring.rst

```diff
@@ -1,108 +1,210 @@
-
 *************
 Configuration
 *************
 
-You can configure different parts of the theme.
-
-Project-wide Configuration
-==========================
+Theme options
+=============
 
-HTML Theme Options
-------------------
+The following options can be defined in your project's ``conf.py`` file, using
+the :confval:`html_theme_options <sphinx:html_theme_options>` configuration option.
 
-The theme's project-wide options are defined in the ``sphinx_rtd_theme/theme.conf``
-file of this repository, and can be defined in your project's ``conf.py`` via
-``html_theme_options``. For example:
+For example:
 
 .. code:: python
 
     html_theme_options = {
-        'canonical_url': '',
+        'canonical_url': 'https://example.com/en/latest/',
         'analytics_id': '',
         'logo_only': False,
         'display_version': True,
         'prev_next_buttons_location': 'bottom',
         'style_external_links': False,
         'vcs_pageview_mode': '',
-        # Toc options
         'collapse_navigation': True,
         'sticky_navigation': True,
         'navigation_depth': 4,
         'includehidden': True,
         'titles_only': False
     }
 
-The following options are available:
+Table of contents options
+-------------------------
+
+The following options affect the `table of contents configuration options`_
+Sphinx uses to generate the documentation navigation.
+
+.. confval:: collapse_navigation
+
+    :type: boolean
+    :default: ``False``
```

Comment:
The default is True https://github.com/rtfd/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/theme.conf#L9